tailcfg/tailcfg.go

```diff
@@ -128,7 +128,8 @@ type CapabilityVersion int
 //   - 85: 2024-01-05: Client understands MaxKeyDuration
 //   - 86: 2024-01-23: Client understands NodeAttrProbeUDPLifetime
 //   - 87: 2024-02-11: UserProfile.Groups removed (added in 66)
-const CurrentCapabilityVersion CapabilityVersion = 87
+//   - 88: 2024-03-05: NodeAttrSuggestExitNode
```

Comment:
`Client understands NodeAttrsSuggestExitNode`

also there is an extra `s` in NodeAttr**s**SuggestExitNode. As it is not being used anywhere, we should make it conform with all the other `NodeAttr*` consts